changelog.txt

```diff
@@ -5,10 +5,16 @@ Changelog
 The **signac-flow** package follows `semantic versioning <https://semver.org/>`_.
 The numbers in brackets denote the related GitHub issue and/or pull request.
 
+Version 0.21
```

Comment:
This is usually added _after_ the release, so that 0.20.0 doesn't refer to 0.21.0 in the changelog.